src/css/text-viewer.css

```diff
@@ -48,7 +57,7 @@ pre.box {
 }
 
 pre code.code-box {
-    min-height: 100%;
+    min-height: 100vh;
```

Comment:
Won't this force a vertical scrollbar to appear when there is only a horizontal one?

Reply:
No, it just makes the code box at least the height of the viewport. No scrollbar showed in my testing as it doesn't overflow the viewport.

Reply:
In the specific case that the text has few lines, but some lines are very long, a horizontal scrollbar should show up, but there should be no vertical scrollbar. However, if `100vh` is used, a vertical one does show up:

![image](https://user-images.githubusercontent.com/26528805/88823900-9f3a0980-d193-11ea-9a17-d6a8fd32dc78.png)

Reply:
Hmm I hadn't tested that. It's weird it does that. Where's the extra height from?

Reply:
It's not just on mobile, the `padding-top` line is before the `padding` line, so it gets overridden. The extra height is because `100vh` uses the *entire height of the viewport*, which includes scrollbars. This means that the `min-height` gets set to the height of the page + the scrollbar height. Keeping it at `100%` should work fine.

Reply:
With 100% the body background was the bootstrap-dark background instead of the code viewer background. Does it show the same for you?

Reply:
You're right, it does. That's really strange, since it didn't have that problem before this PR. Here's an example of my uploader which is running v2.0.0: https://img.vashevko.com/test1.txt

Maybe some changes to the bootstrap CSS changed something? I'm not sure, but I'll look into how to fix this without using `100vh`.